.flake8

```diff
@@ -29,4 +29,4 @@ import-order-style = pycharm
 application-import-names =
     y0
     tests
-format = ${cyan}%(path)s${reset}:${yellow_bold}%(row)d${reset}:${green_bold}%(col)d${reset}: ${red_bold}%(code)s${reset} %(text)s
```

Comment:
is there a reason for this change?

Reply:
In the terminal, the character number was much more visible than the line-number.  This change made the line-number more emphasized. 

These days I'm actually using `format = ${red}%(code)s${reset} ${cyan}%(path)s${reset}:${red_bold}%(row)d${reset} %(text)s`.  Which doesn't give the character number at all and moves the error-code to the left (where it is easier to ignore because its rarely needed).